Moodis/Moodis/Ui/MenuForm.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using moodis;
+using Moodis.Network.Face;
+
+namespace Moodis.Ui
```

Comment:
Move MenuForm and MenuMethods to Feature folder, so the project structure would look like this: Feature -> Menu -> MenuForm, MenuMethods.